src/test/java/io/cdap/ArtifactPackagerMojoTest.java

```diff
@@ -27,6 +44,8 @@ public void execute_relativeOutputDirSpecified_andFilesExistInBuildDir_Artifacts
 
     String baseDir = PlexusTestCase.getBasedir();
     File testPom = new File(baseDir, "src/test/resources/test_poms/artifactPackager-pom.xml");
+    assertNotNull( testPom );
```

Comment:
style nit: don't need the whitespace around the method call -- `assertNotNull(testPom);`